build.gradle

```diff
@@ -104,6 +105,43 @@ switch (JavaVersion.current()) {
         break;
 }
 
+task cloneAndBuildJspecify {
+    description 'Obtain or update JSpecify.'
+    doLast {
+        if (JavaVersion.current() > JavaVersion.VERSION_1_8) {
+            if (file(jspecifyHome).exists()) {
```

Comment:
Can't you use the `createCloneTask` to do this?

Reply:
ok, used createCloneTask to simplify the task.